start/sw/gui-app/electron.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron
+subsection: gui-app
+section: start-sw
+description: 
+---
+
+# Create an application using Electron
+
+[Electron][0] is cross platform framework for creating desktop applications using
+web technologies. It is developed by GitHub and released under the MIT license.
```

Comment:
I'd skip the last sentence and would prefer some usual / suitable use-cases, or some example, WDYT?